src/main/java/equipments/technology/TechnologyEquipmentFactory.java

```diff
@@ -0,0 +1,13 @@
+package equipments.technology;
+
+public class TechnologyEquipmentFactory {
+    public TechnologyEquipment getTechnologyEquipment(String technologyEquipmentType, String IP){
+        if(technologyEquipmentType.equalsIgnoreCase("Voip"))
```

Comment:
for values of a predefined list, it's better to use `enum`s instead of `string`s. a string is very fragile (a small typo could cause a bug)